app/views/movies/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @movie.title %></h1>
+<p>Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+<% unless @movie.actors.empty? %>
+  <% @movie.actors_by_age.each do |actor| %>
+    <section id="actor-<%= actor.id %>">
+      <h4><%= actor.name %></h4>
+      <p>Age: <%= actor.age %></p>
+    </section>
+  <% end %>
+<% end %><br/>
+
+<p>Average Age of Actors: <%= @movie.average_age_of_actors.round %></p><br/>
+
+<form action="/movies/<%= @movie.id %>" method="post">
+  <label for="actor_id">Add an Actor to this Movie by Actor ID</label>
+  <input type="number" name="actor_id" id="actor_id">
+  <input type="submit" value="Submit">
```

Comment:
Try to start working yourself into the habit of using Rails' form helpers. It's a much more conventional and common way you'll see forms built in Rails apps.